nselib/msrpctypes.lua

```diff
@@ -4523,6 +4523,26 @@ function unmarshall_SERVICE_STATUS(data, pos)
 end
 
 
+--- Unmarshalls a null-terminated Unicode string based upon a 32-bit offset (LPTSTR)
+-- @param arguments The data being processed
```

Comment:
Why is it called `arguments`? Perhaps something like `w_str` would be more representative?